macos.md

```diff
@@ -191,9 +191,8 @@ Make sure that you're running the 2nd-newest macOS version or the newest version
 
     ```bash
     [[ -d /opt/homebrew/var/postgresql@16 ]] && PGDATA_TMP=/opt/homebrew/var/postgresql@16 || PGDATA_TMP=/usr/local/var/postgresql@16
-    echo "\nexport PGDATA=$PGDATA_TMP\nexport PSQL_PAGER=\"less --chop-long-lines --header 1\"" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshrc' || echo '.bash_profile'`
+    echo -e "export PGDATA=$PGDATA_TMP\nexport PSQL_PAGER=\"less --chop-long-lines --header 1\"\nexport LANG=en_US.UTF-8" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshrc' || echo '.bash_profile'`
```

Comment:
probably we should keep the `\n` at the beginning, in case the config file doesn't have a newline at the end of the file

Reply:
Keeping the `\n` without the `-e` flag works in macOS but not on `Windows` and `Linux` because new line is handled differently on those systems

```bash
echo "\n" vs echo -e "\n"
```
<img width="1920" alt="Screenshot 2024-08-07 at 10 42 16" src="https://github.com/user-attachments/assets/25db5f99-2eb0-4cc5-b7e7-7259078cec95">

![Screenshot 2024-08-07 104017](https://github.com/user-attachments/assets/0d5cbd11-cc82-43a3-a802-ea464437801f)


These commands produce different results on macOS than on others
macOS
```bash
echo "\nexport PGDATA=$PGDATA_TMP\nexport PSQL_PAGER=\"less --chop-long-lines --header 1\"\nexport LANG=en_US.UTF-8" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshrc' || echo '.bash_profile'` 
```
Result
```bash
export PGDATA=/opt/homebrew/var/postgresql@16
export PSQL_PAGER="less --chop-long-lines --header 1"
export LANG=en_US.UTF-8
```

Linux
```bash
echo "\nexport PSQL_PAGER=\"less --chop-long-lines --header 1\"\nexport LANG=en_US.UTF-8" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshenv' || echo '.bashrc'` 
```

Result
```bash
\nexport PSQL_PAGER="less --chop-long-lines --header 1"\nexport LANG=en_US.UTF-8
```

Windows
```bash
echo "\nexport PSQL_PAGER=\"less --chop-long-lines --header 1\"\nexport LANG=en_US.UTF-8" >> "$USERPROFILE/.bash_profile"
```

Result
```bash
\nexport PSQL_PAGER="less --chop-long-lines --header 1"\nexport LANG=en_US.UTF-8
```

We can add the `\n` only for the macOS script
db38259f853782a9d12dfa9e015991f195be81b8

Reply:
If I want to use the first `\n` on other systems, I can do something like this, but I don't see the need apart from making everything look the same. We can have the new line for macOS and have the `-e` for other systems

macOS
```bash
echo -e "\nexport PGDATA=$PGDATA_TMP\nexport PSQL_PAGER=\"less --chop-long-lines --header 1\"\nexport LANG=en_US.UTF-8" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshrc' || echo '.bash_profile'` 
```
Linux
```bash
echo -e "\nexport PSQL_PAGER=\"less --chop-long-lines --header 1\"\nexport LANG=en_US.UTF-8" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshenv' || echo '.bashrc'` 
```
Windows
```bash
echo -e "\nexport PSQL_PAGER=\"less --chop-long-lines --header 1\"\nexport LANG=en_US.UTF-8" >> "$USERPROFILE/.bash_profile"
```

Reply:
As agreed in the daily, we are now using the version with `-e` without the first `\n` for all systems